collections_hierarchy/mixins.py

```diff
@@ -60,29 +112,37 @@ def __init__(self, initial=None):
 
 class OperableMixin(object):
     def __add__(self, other):
-        pass
+        new_obj = object.__new__(self.__class__)
```

Comment:
Interesting way to make a new object with the same type... I couldn't even imagine it.